images/installer-origin-release/Dockerfile

```diff
@@ -1,12 +0,0 @@
-# This Dockerfile is used to create the image quay.io/coreos/origin-release:bazel used in CI
```

Comment:
Note to self, this `Dockerfile` ended up [here][1] in openshift/release#1317, so we don't need it here.

[1]: https://github.com/openshift/release/blob/c6b9807b873f9be87f438e3328e3a20358afb967/projects/origin-release/bazel/Dockerfile